samples/test/vision_classification_deploy_model_node_count.test.js

```diff
@@ -20,40 +20,31 @@ const {AutoMlClient} = require('@google-cloud/automl').v1;
 
 const cp = require('child_process');
 
-const execSync = cmd => cp.execSync(cmd, {encoding: 'utf-8'});
+const fs = require('fs');
+
+const tempFile = 'temp.txt';
+var stderr_output = fs.createWriteStream(tempFile);
+const execSync = cmd => cp.execSync(cmd, {encoding: 'utf-8', stdio: ['pipe', 'pipe', stderr_output] });
```

Comment:
@bcoe, you might know a better way to do this than I in Node.js

To speed up the tests, I'm going to ask the api to deploy a model that doesn't exist, the request is valid, but when the api tries to deploy the model it throws an error that the model does not exist. So we can still test the sample and get a fast test. 

But for the child process, I'm piping the stderr output to a file and reading that to verify the error is what I want. 

Thoughts?

Reply:
Rather than piping to a file, I think you could use `spawnSync` which provides both `stdout` and `stderr`, I think that would feel less weird.

Reply:
Updated